packages/frontend/src/retro/hooks/useTimer.ts

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useMemo, useState } from "react";
+
+interface useTimerProps {
+  timerFinished: () => void;
```

Comment:
At least in this project, we use the words `on...` and `handle...` when defining callbacks for button clicks etc.

When we actively define what we do, we use handle, e.g. `handleIncrementMinutes`.
When providing an option to set this value through a callback, we use on, e..g. `onIncrementMinutes`.

In this case, we would call this `onTimerFinished`.
When setting a value to this property later in the code, it would look like `onTimerFinished={handleTimerFinished}`.
You'll find that a lot throughout this code. It really helps to understand whether or not something is given or requested.

Reply:
I see makes the intention clearer. I fixed it here and found no other place to be fixed

Reply:
I double checked in the code and we usually use `onTimerFinish` and not `onTimerFinished`. Also react provides `onChange` and `onClick` properties.

Reply:
Ok i renamed it to onTimerFinish. What do you mean with the onChange and onClick properties ?